src/prettytable/prettytable.py

```diff
@@ -74,6 +76,61 @@ def _get_size(text):
     return width, height
 
 
+##############################
+# ALIGNMENT VALIDATORS       #
+##############################
+
+
+def validate_align(val):
+    try:
+        assert val in ["l", "c", "r"]
+    except AssertionError:
+        raise ValueError(f"Alignment {val} is invalid, use l, c or r")
+
+
+def validate_valign(val):
+    try:
+        assert val in ["t", "m", "b", None]
+    except AssertionError:
+        raise ValueError(f"Alignment {val} is invalid, use t, m, b or None")
+
+
+def validate_header_align(val):
+    try:
+        assert val in ["l", "c", "r", None]
+    except AssertionError:
+        raise ValueError(f"Alignment {val} is invalid, use l, c, r or None")
+
+
+# A dict extension to manage field alignment. Used to:
+# Have global alingment sepreate from field alignments.
+# Validate values when setting for individual fields
+class Alignment(dict):
```

Comment:
Also private/internal?
```suggestion
class _Alignment(dict):
```